docs/Installation.md

```diff
@@ -0,0 +1,80 @@
+Before deploying the tooling, you must have the following prepared
+
+* An OpenShift 4.9 or higher Environment
+* A machine from which to run the install (usually your laptop)
+  * The OpenShift Command Line Tool (oc)
+  * yq
+  * jq
+  * git
+  * Github Token with the following permissions
+    * `repo`
+    * `delete_repo`
+  * Valid quay token where to push AppStudio components images generated by the e2e framework
+
+## Initial Deployment Steps
+
+Before starting the deployment steps of appstudio in e2e mode in Openshift CI you should skip the first 2 steps
+
+1. Access To Openshift Cluster.
+
+2. Login to the cluster as `admin`
+
+   ```bash
+    oc login -u <user> -p <password> --server=<oc_api_url>
+   ```
+
+3. Install Red Hat App Studio in e2e mode. The e2e framework by default will use the `redhat-appstudio-qe` github organization by default. If u want to change the github org you should define `GITHUB_E2E_ORGANIZATION` environment with your custom github organization
+
+   ```bash
+      # In Openshift CI there are some example about how to use the installation script. See infra-deployments script https://github.com/redhat-appstudio/application-service/blob/main/.ci/oci-e2e-has.sh#L59
+      $ROOT_DIR/scripts/install-appstudio-e2e-mode.sh install
+   ```
+
+4. Compile the e2e tests
+
+   ```bash
+    make build
+   ```
+
+5. Run the e2e tests
+
+   ```bash
+    `$ROOT_DIR/bin/e2e-appstudio`
+   ```
+
+Where are:
+
+- `install` - Flag to indicate the installation. If the flag will not be present you can `source` the script and use the bash functions.
+
+The following environments are used to launch the Red Hat AppStudio installation in e2e mode and the tests execution:
+
+| Variable | Required | Explanation | Default Value |
+|---|---|---|---|
+| `GITHUB_TOKEN` | yes | A github token used to create AppStudio applications in github  | ''  |
+| `QUAY_TOKEN` | yes | A quay token to push components images to quay.io. Note the quay token must be in base 64 format `ewogI3dJhdXRocyI6I...` | '' |
+| `GITHUB_E2E_ORGANIZATION` | no | GitHub Organization where to create/push Red Hat AppStudio Applications  | `redhat-appstudio-qe`  |
+| `QUAY_E2E_ORGANIZATION` | no | Quay organization where to push components containers | `redhat-appstudio-qe` |
+| `E2E_APPLICATIONS_NAMESPACE` | no | Name of the namespace used for running HAS E2E tests | `appstudio-e2e-test` |
+
+## Install e2e binary in openshift-ci and use pairing PRs feature
+
+The e2e tests are executed against almost all App Studio repositories.
+
+Sometimes when we have changes in App Studio we are introducing some breaking changes and the e2e will fail. To prevent this the e2e framework installation in openshift-ci support a new feature of pairing the PR opened against an App Studio repository with the e2e forks based in branch names. Before the e2e framework will be executed in openshift-ci, the logic automatically tries to pair a PR opened in some repo with a branch of the same name that
```

Comment:
Seems to me this paragraph and the paragraph starting on line 72 are saying basically the same... Maybe you intended to rewrite it and forgot to delete the old one? ;-)